javascript/advanced_examples/gdx_start_stop_desmos.html

```diff
@@ -20,53 +12,53 @@
   
   <!--include the Desmos and the GoDirect libraries-->
   <script src="https://unpkg.com/@vernier/godirect/dist/godirect.min.umd.js"></script>
-  <meta name="viewport" content="width=device-width, initial-scale=1">
   <script src="https://www.desmos.com/api/v1.3/calculator.js?apiKey=dcb31709b452b1cf9dc26972add0fda6"></script>
-  
-<!--style the webpage with css (optional)-->
-<style>
-   .button {
-  background-color: green;
-  border: none;
-  color: white;
-  padding: 15px 32px;
-  text-align: center;
-  text-decoration: none;
-  display: inline-block;
-  font-size: 16px;
-  margin: 4px 2px;
-  cursor: pointer;
-  }
-  div.fixed {
-  position: fixed;
-  top: 0;
-  right: 0;
-  width: 300px;
-  border: 3px solid #73AD21;
-}
-</style>
-  
+
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <!--style the webpage with css (optional)-->
+  <style>
+    .button {
+      background-color: green;
+      border: none;
+      color: white;
+      padding: 15px 32px;
+      text-align: center;
+      text-decoration: none;
+      display: inline-block;
+      font-size: 16px;
+      margin: 4px 2px;
+      cursor: pointer;
+    }
+    div.fixed {
+      position: fixed;
+      top: 0;
+      right: 0;
+      width: 300px;
+      border: 3px solid #73AD21;
+    }
+  </style>  
 </head>
 
 <body>  
- <!--format the webpage -->
- <div align="left"> 
+  <h1>Vernier Go Direct<sup>&reg</sup> JavaScript Advanced Example</h1>
+	<p>
+		This example shows how you can use the <a href="https://www.npmjs.com/package/@vernier/godirect">Vernier Go Direct Module</a> to connect to a <a href="https://www.vernier.com/products/sensors/go-direct-sensors/">Vernier Go Direct Sensor</a> and put data into Desmos.
+	</p>
+
+  <div align="left"> 
```

Comment:
We'll need to eventually strip these kinds of things out, since the align attribute was deprecated in favor of css long ago